test/functional/wallet_basic.py

```diff
@@ -355,7 +355,7 @@ def run_test(self):
         for mode in [True, False]:
             self.nodes[0].ensure_ascii = mode
             # unicode check: Basic Multilingual Plane, Supplementary Plane respectively
-            for s in [u'рыба', u'𝅘𝅥𝅯']:
+            for s in [u'рыба', u'𝅘𝅥𝅯']:
```

Comment:
Please revert this change. It looks accidental.

Reply:
Hopefully fixed.

Reply:
No, the change is still there. You can check it on https://github.com/bitcoin/bitcoin/pull/12593/files after pushing your latest version.

Reply:
This was very weird. It's some strange character encoding that my machine somehow automatically converts when the file is opened. I did several attempts to fix it using the command line and different text editors and I think I did it...